keyboards/era/linx3/fave65s/keyboard.json

```diff
@@ -0,0 +1,197 @@
+{
+    "manufacturer": "eerraa",
+    "keyboard_name": "FAve65S",
+    "maintainer": "eerraa",
+    "bootloader": "rp2040",
+    "build": {
+        "debounce_type": "sym_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "cols": ["GP28", "GP27", "GP26", "GP0", "GP1", "GP2", "GP3", "GP4", "GP5", "GP6", "GP7", "GP8", "GP9", "GP12", "GP11", "GP10"],
+        "rows": ["GP25", "GP29", "GP18", "GP23", "GP24"]
+    },
+    "processor": "RP2040",
+    "rgb_matrix": {
+        "animations": {
+            "alphas_mods": true,
+            "gradient_up_down": true,
+            "gradient_left_right": true,
+            "breathing": true,
+            "band_sat": true,
+            "band_val": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_up_down": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "rainbow_moving_chevron": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "rainbow_pinwheels": true,
+            "flower_blooming": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "pixel_fractal": true,
+            "pixel_flow": true,
+            "pixel_rain": true,
+            "starlight": true,
+            "starlight_dual_hue": true,
+            "starlight_dual_sat": true,
+            "riverflow": true
+        },
+        "default": {
+            "animation": "rainbow_moving_chevron"
+        },
+        "driver": "ws2812",
+        "layout": [
+            {"matrix": [2, 0], "x": 0, "y": 0, "flags": 8},
+            {"x": 0, "y": 0, "flags": 2},
```

Comment:
As per https://allthatkeyboard.com/product/171893, these don't appear to all be underglow LEDs. Each should be assigned the appropriate matrix position and the flag parameter updated.

Reply:
Thank you for your kind cross-checking.

However, This is a newly designed PCB.
Since only the Hotswap version existed previously, Linworks created a new solder version.
The new solder version has a caps lock indicator and underglow RGB.

Reply:
So, is it okay if I change this comment to Resolved?

Reply:
Are all 40+ LEDs underglow LEDs?

Reply:
Yes, it has 44 2020size underglow RGB and 1 3535size indicator RGB.
A total of 45 RGB elements are used.

Reply:
Understood. I see that the 's' in keyboard name probably stands for solder, but if something could be added to the readme too, that'd be helpful. This can be marked as resolved.

Reply:
:) Thank you for your help.